tests/test-cases/include-component/integration.include-component.test.ts

```diff
@@ -0,0 +1,70 @@
+import {WriteStreamsMock} from "../../../src/write-streams";
+import {handler} from "../../../src/handler";
+import assert, {AssertionError} from "assert";
+import {initSpawnSpy} from "../../mocks/utils.mock";
+import {WhenStatics} from "../../mocks/when-statics";
+
+beforeAll(() => {
+    initSpawnSpy(WhenStatics.all);
+});
+
+test.concurrent("include-component no component template file (http)", async () => {
+    initSpawnSpy([WhenStatics.mockGitRemoteHttp]);
```

Comment:
can we setup a dummy bot account with ssh key for the runner, 

too much mocking needs to be done for the `protocol: git`

Reply:
Can you elaborate, what exactly you mean? (I'm not following)

Reply:
I mean setting up a ssh key for the runner so we that we can run command like 
`git archive --remote=ssh://git@gitlab.com/components/go.git master main.go`

without having the need to mock

Reply:
Aaah, the Github Actions runner :smile: